app/controllers/ajax/report_controller.rb

```diff
@@ -11,35 +13,19 @@ def create
       return
     end
 
-    unless %w(answer comment question user).include? params[:type]
-      @response[:message] = t(".unknown")
-      return
+    result = UseCase::Report::Create.call(
+      reporter_id: current_user.id,
+      object_id:   params[:id],
+      object_type: params[:type],
+      reason:      params[:reason],
+    )
+
+    if result[:status] == 201
```

Comment:
I don't think this if/else condition is necessary

`result[:status]` will always be `201`, and the errors raised from the UseCase are caught in the base class https://github.com/Retrospring/retrospring/blob/b4929300fc1923929a679157f6bb27c36b425c23/app/controllers/ajax_controller.rb#L84-L94